From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: David Hongseock Kim <hongpooh.kim@samsung.com>
Date: Sat, 8 Aug 2020 13:58:25 +0900
Subject: NOUPSTREAM: ANDROID: pcie/dwc: driver: export dw_handle_msi_irq

Needed for vendors' pcie stack.

[CPNOTE: 06/07/21] Lee: Vendor GKI code

Bug: 165922442
Signed-off-by: David Hongseock Kim <hongpooh.kim@samsung.com>
Change-Id: I900b5a55b174819573c2f8eedcefae73aee7b6a7
Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Signed-off-by: Will McVicker <willmcvicker@google.com>
Signed-off-by: Lee Jones <joneslee@google.com>
---
 drivers/pci/controller/dwc/pcie-designware-host.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/dwc/pcie-designware-host.c b/drivers/pci/controller/dwc/pcie-designware-host.c
--- a/drivers/pci/controller/dwc/pcie-designware-host.c
+++ b/drivers/pci/controller/dwc/pcie-designware-host.c
@@ -82,6 +82,7 @@ irqreturn_t dw_handle_msi_irq(struct dw_pcie_rp *pp)
 
 	return ret;
 }
+EXPORT_SYMBOL_GPL(dw_handle_msi_irq);
 
 /* Chained MSI interrupt service routine */
 static void dw_chained_msi_isr(struct irq_desc *desc)
